templates/loki-alert-rule-group-composite-controller.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: metacontroller.k8s.io/v1alpha1
+kind: CompositeController
+metadata:
+  name: {{ .Values.appName }}
```

Comment:
I've usually used .Release.Name. is this a better approach?

Reply:
I haven't used that before - I'll make that update

Reply:
Yeah let me know if you come across something that says we shouldn't use it. I did it for strimzi and it seems to be working. Minus we haven't tested any upgrade paths thoroughly yet